experiments/modelfree/plot_noisy_observations.py

```diff
@@ -0,0 +1,141 @@
+import json
+import math
+import os
+
+from matplotlib import pyplot as plt
+import pandas as pd
+import seaborn as sns
+
+RESULT_DIR = "/Users/cody/Data/adversarial_policies/ray_results/noisy_victim_obs/"
+OUT_DIR = "/Users/cody/Data/adversarial_policies/noisy_obs_plots/results_may_22"
+env_lookup = {
```

Comment:
Consider using `modelfree.visualize.util.PRETTY_ENV` instead. (The key-values are the other way around, but it's easy to reverse if you need.)